module-system/sov-modules-core/src/storage/scratchpad.rs

```diff
@@ -449,6 +460,88 @@ impl<'a, C: Context> StateReaderAndWriter for AccessoryWorkingSet<'a, C> {
     }
 }
 
+/// Provides specialized working set wrappers for dealing with protected state.
+pub mod kernel_state {
+    use sov_rollup_interface::da::DaSpec;
```

Comment:
I added this issue #1166 and made this PR #1164 in order to bring more tests to `sov-modules-core`. Let;s add some tests for  `VersionedWorkingSet` there?

Reply:
Good idea. Will add.

Reply:
Done